### PR TITLE
Adds a non phallic, white hardsuit for the heads

### PR DESCRIPTION
**Pull request by Kmc2000**
^front

^back

^helm

NOTICE:

They aren't included in suit storage units, and i haven't replaced the ones on station, they're here for review and if people like em enough then i'll add a few to EVA to replace the standard ones (the command eva section).

Feedback welcome

🆑 Kmc2000
rscadd: adds a hardsuit for the better endowed members of the crew
/🆑

### CODE REVIEW

File: code/modules/clothing/spacesuits/hardsuit.dm [hunk 1]
```diff
@@ -618,3 +618,19 @@
...
 	item_color = "mmedic"
 
+
+
+	//Heads of staff hardsuit
+/obj/item/clothing/head/helmet/space/hardsuit/heads
+	name = "command hardsuit helmet"
+	desc = "A sleek, white hardsuit helmet used by the heads of staff, the stitching is embossed with paper from last month's paycheck for the crew"
+	icon_state = "hardsuit0-heads"
+	armor = list(melee = 30, bullet = 5, laser = 10, energy = 5, bomb = 30, bio = 100, rad = 50)
```
Comment (Super3222) [inline code/modules/clothing/spacesuits/hardsuit.dm]: Remove the rad protection.

Comment (Super3222) [inline code/modules/clothing/spacesuits/hardsuit.dm]: Or lower it to 20 like the EVA version.

**Comment by Kmc2000** [comment]
@Super3222  rad protection is standard on all hardsuits though innit?

File: code/modules/clothing/spacesuits/hardsuit.dm (hunk 1 above)
Comment (Stealthkibbler) [inline code/modules/clothing/spacesuits/hardsuit.dm]: RADICAL

**Comment by Kmc2000** [comment]
Do not merge, fixing some bugs

**Comment by Kmc2000** [comment]
Ech screw it, it looks weird in game and i can't get it to work

RIP

**Pull request closed without merging**